GetRawYieldsDplusDs.C

```diff
@@ -432,6 +478,16 @@ int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfile
                 hRawYieldsFracGaus2->SetBinContent(iPt+1,frac2gaus);
                 hRawYieldsFracGaus2->SetBinError(iPt+1,frac2gauserr);
             }
+            else if(SgnFunc[iPt]==AliHFInvMassFitter::k2GausSigmaRatioPar) {
+                double sigmaRatio1_2 = massFunc->GetParameter(parSigmaRatio);
+                double sigmaRatio1_2err = massFunc->GetParError(parSigmaRatio);
+                double sigma2 = sigma/sigmaRatio1_2;
+                double sigma2err =0; //todo: add error propagation (correlation?)
```

Comment:
the correlation should be taken from the covariance of the fit, but is not necessary, you can just consider the two uncorrelated